soa/utils/scope.h

```diff
@@ -1,4 +1,4 @@
-/* scope.h
```

Comment:
We probably want these changes in master so I'd consider cherry-picking the commit.